diffus-derive-test/src/lib.rs

```diff
@@ -4,7 +4,36 @@
 mod test {
     use diffus_derive::Diffus;
 
-    use diffus::{self, Diffable};
+    use diffus::{self, edit, Diffable};
+
+    mod hide {
+        use super::*;
+
+        #[derive(Diffus)]
+        pub struct Inside {
+            pub p: u32,
+        }
+
+        impl Inside {
+            pub fn new(p: u32) -> Inside {
+                Inside { p }
+            }
+        }
+
+    }
+
+    #[test]
+    fn vis_check() {
+        if let edit::Edit::Change(diff) = hide::Inside::new(0).diff(&hide::Inside::new(1)) {
```

Comment:
You should be able to unpack the below row already in here since it's a simple field access, removing one of the if/else?

Reply:
correct, played around with it being private, forgot that I settled for having it public in the tests